core/src/main/java/tc/oc/pgm/map/includes/MapIncludeProcessorImpl.java

```diff
@@ -0,0 +1,107 @@
+package tc.oc.pgm.map.includes;
+
+import com.google.common.collect.Sets;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Logger;
+import org.jdom2.Document;
+import org.jdom2.Element;
+import org.jdom2.JDOMException;
+import org.jdom2.input.SAXBuilder;
+import tc.oc.pgm.api.Config;
+import tc.oc.pgm.api.map.exception.MapMissingException;
+import tc.oc.pgm.api.map.includes.MapInclude;
+import tc.oc.pgm.api.map.includes.MapIncludeProcessor;
+import tc.oc.pgm.util.xml.InvalidXMLException;
+import tc.oc.pgm.util.xml.SAXHandler;
+import tc.oc.pgm.util.xml.XMLUtils;
+
+public class MapIncludeProcessorImpl implements MapIncludeProcessor {
+
+  private final Logger logger;
+  private final Set<MapInclude> includes;
```

Comment:
```suggestion
  private final Map<String, MapInclude> includes;
```
Keep them in a hash map indexed by id, after all, that's the way you're going to be accessing them